Release Notes/2017/2017-10-24-cloud-platform-release-notes.md

```diff
@@ -0,0 +1,32 @@
+{{{
+"title": "Cloud Platform - Release Notes: October 24, 2017",
+"date": "10-24-2017",
+"author": "Matthew Ordman",
+"attachments": [],
+"contentIsHTML": false
+}}}
+
+### New Features (1)
+* __Cloud Application Manager (CAM)__
+
+  __Monitoring Enhancements__
+
+  Cloud Application Manager's monitoring service, Watcher, has been integrated with CenturyLink's Analytics service.  The integration of Watcher with Analytics provides customers the added capabilities of advanced alerting based on analytic predictive models providing event correlation against arbitrary time series streams.
+
+  __Cloud Optimization and Analytics__
+
+  Cloud Application Manager customers who utilize Microsoft Azure through CenturyLink for Consolidated Billing and Platform Support now may generate multiple [Azure subscriptions](https://www.ctl.io/knowledge-base/cloud-application-manager/cloud-optimization/partner-cloud-integration-azure-new/). Multiple Azure subscriptions provides your administrators the ability to easily manage users and resources for multiple Azure projects and departments by placing them in different Azure "accounts."
```

Comment:
Change the link to point to the file: `../../Cloud Application Manager/Cloud Optimization/partner-cloud-integration-azure-new.md`